dataland-frontend/tests/e2e/support/Commands.ts

```diff
@@ -18,7 +19,12 @@ export function visitAndCheckAppMount(endpoint: string): Chainable<JQuery> {
   return cy.visit(endpoint).get("#app").should("exist");
 }
 
+export function deleteDownloadsFolder(): Chainable<void> {
```

Comment:
Ist das jetzt nicht doppelt definiert? Die Funktion gab es doch auch im config file schon.

Reply:
Also es ist nicht direkt doppelt definiert sondern wie folgt:
- In der ```cypress.config.ts``` wird eine generelle cypress task eingeführt, die Ordner löschen kann, deren Pfad man als input param reingibt.
- Im ```Commands.ts``` wird eine cypress command definiert, welche die ```deleteFolder``` task explizit dafür nutzt, den downloads folder zu löschen.

Da wir aber in den Tests aktuell eigentlich nur einmal den downloads folder löschen müssen, ist die allgemein gehaltene ```deleteFolder``` task eigentlich nicht notwendig, bzw. man könnte sie direkt so schreiben, dass sie den downloadsFolder löscht, und entsprechedn umbenennen.

Soll ich das machen? Was meinst du?

Reply:
Ja wenn das nur einmal benutzt wird, ist es etwas viel overhead.